src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -3,77 +3,77 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
 
-    public static class LanguageServerConstants
```

Comment:
Recommend viewing with "?w=1", there was some formatting that still needed to happen after the namespace change.